app/views/comments/_comment_date.erb

```diff
@@ -4,4 +4,10 @@
       <%= decorated_comment.readable_publish_date %>
     </time>
   </a>
-</div>
+  <% if !decorated_comment.edited_at.blank? %>
```

Comment:
this could just be `if decorated_comment.edited_at`

Reply:
Or `if  decorated_comment.edited_at.present?` if we want to stick with an explicit boolean method.